diff --git a/node_modules/@expo/server/build/index.js b/node_modules/@expo/server/build/index.js
index 6ca1f81..8bf817f 100644
--- a/node_modules/@expo/server/build/index.js
+++ b/node_modules/@expo/server/build/index.js
@@ -102,9 +102,20 @@ function createRequestHandler(distFolder, { getRoutesManifest: getInternalRoutes
             routesManifest = getRoutesManifest(distFolder);
         }
         const url = new url_1.URL(request.url, 'http://expo.dev');
-        const sanitizedPathname = url.pathname;
+        let sanitizedPathname = url.pathname;
         debug('Request', sanitizedPathname);
         if (request.method === 'GET' || request.method === 'HEAD') {
+
+            // TODO: Official redirect support
+            if (sanitizedPathname.match(/^\/expo\/showcase/)) {
+                return new Response('Redirecting', {
+                    status: 301,
+                    headers: {
+                        Location: '/blog/expo-2024',
+                    },
+                });
+            }
+
             // First test static routes
             for (const route of routesManifest.htmlRoutes) {
                 if (!route.namedRegex.test(sanitizedPathname)) {
