diff --git a/node_modules/@expo/metro-runtime/build/error-overlay/toast/ErrorToastContainer.web.js b/node_modules/@expo/metro-runtime/build/error-overlay/toast/ErrorToastContainer.web.js
index 66f1d3b..2b6fcd8 100644
--- a/node_modules/@expo/metro-runtime/build/error-overlay/toast/ErrorToastContainer.web.js
+++ b/node_modules/@expo/metro-runtime/build/error-overlay/toast/ErrorToastContainer.web.js
@@ -75,7 +75,7 @@ const styles = react_native_1.StyleSheet.create({
         bottom: 6,
         left: 10,
         right: 10,
-        position: "absolute",
+        position: "fixed",
         maxWidth: 320,
     },
 });
