diff --git a/node_modules/expo-router/src/static/renderStaticContent.tsx b/node_modules/expo-router/src/static/renderStaticContent.tsx
index 7fdae39..702f2e5 100644
--- a/node_modules/expo-router/src/static/renderStaticContent.tsx
+++ b/node_modules/expo-router/src/static/renderStaticContent.tsx
@@ -7,6 +7,7 @@
 
 import { ServerContainer, ServerContainerRef } from "@react-navigation/native";
 import App, { getManifest } from "expo-router/_entry";
+import * as Font from 'expo-font/build/server';
 import Head from "expo-router/head";
 import React from "react";
 import ReactDOMServer from "react-dom/server";
@@ -45,6 +46,11 @@ export function getStaticContent(location: URL): string {
   // "Warning: Detected multiple renderers concurrently rendering the same context provider. This is currently unsupported."
   resetReactNavigationContexts();
 
+  // Clear any existing static resources from the global scope to attempt to prevent leaking between pages.
+  // This could break if pages are rendered in parallel or if fonts are loaded outside of the React tree
+  Font.resetServerContext();
+
+
   const html = ReactDOMServer.renderToString(
     <Head.Provider context={headContext}>
       <ServerContainer ref={ref}>
@@ -73,6 +79,11 @@ export function getStaticContent(location: URL): string {
 
   output = output.replace("</head>", `${css}</head>`);
 
+
+  // Inject static fonts loaded with expo-font
+  output = output.replace('</head>', `${Font.getServerResources().join('')}</head>`);
+
+
   return "<!DOCTYPE html>" + output;
 }
 
